packages/dataviews/src/filter-summary.js

```diff
@@ -105,8 +116,17 @@ export default function FilterSummary( { filter, view, onChangeView } ) {
 								key={ element.value }
 								role="menuitemradio"
 								aria-checked={ isActive }
-								prefix={ isActive && <Icon icon={ check } /> }
-								onSelect={ () =>
+								prefix={
+									isActive ? (
+										<Icon icon={ check } />
+									) : (
+										<span
+											className="dataviews__filters-custom-menu-radio-item-prefix"
```

Comment:
> Is this something that could be baked in the components?

I don't think so — it is actually a feature of the component to have its menu items collapse the prefix when possible. Only `DropdownMenuRadioItem` and `DropdownMenuCheckboxItem` have non-collapsing prefixes baked in, because of the check icon.

Unfortunately, since we're using a `DropdownMenuItem` + custom radio semantics, the spacing needs to be added in a custom way too.

I believe that we're going to move away from this solution soon anyway, since it was established that radio menu items semantics is not the best fit for the filters dropdown.